server/public/styles.css

```diff
@@ -0,0 +1,37 @@
+*{
+  margin: 0;
+  box-sizing: border-box;
+}
+header{
+  background-color: rgb(46,51,56);
+  color: white;
+  padding: 0.5rem;
+}
```

Comment:
Bootstrap has color, background, and spacing utilities you can use.

https://getbootstrap.com/docs/4.4/utilities/colors/
https://getbootstrap.com/docs/4.4/utilities/spacing/